generate/writeToJsonFile.js

```diff
@@ -0,0 +1,82 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
+}
+
+
+export function writeJsonArray(arrayNewItems, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
+  // READ FILE
+  fs.readFile(DEST_JSON_FILE, (err, data) => {
+    if (err) {
+      if (err.code === 'ENOENT') {
+        data = JSON.stringify([])
+        fs.writeFileSync(DEST_JSON_FILE, '[]\n')
+      } else {
+        process.exit(1)
+      }
+    }
+    // ALL CONTENT INTO ARRAY
+    const allFileContent = parseReadData(data)
+
+    // UPDATE MATCHING ARRAY ITEMS (ONLY) TO CONTENT
+    arrayNewItems.map(item => {
```

Comment:
Yes you are right. Sorry if you had to spend time on that PR it was not the objective.